src/components/atoms/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { buttonVariants } from "@/components/ui/button";
+import Link from "next/link";
+
+interface LinkButton {
+  title: string;
+  icon?: React.ReactNode;
+  href: string;
+}
+
+const LinkButton: React.FC<LinkButton> = ({ title, icon, href }) => {
+  return (
+    <Link
+      className={buttonVariants({
+        variant: "outline",
+        className: "self-end flex",
+      })}
+      href={href}
+    >
+      {icon}
+      <p className="pl-2">{title}</p>
+    </Link>
```

Comment:
To chyba można zrobić tak: (z odpowiednimi propami oczywiście)

`<Link asChild><Button>...</Button></Link>`